core/models.py

```diff
@@ -11,10 +18,18 @@ def __str__(self):
 
 class SearchQuery(models.Model):
 
+    """
+    Thia model is used to store the search queries in the database.
+    Attributes:
+        query (str): The search query received in API calls (can be null).
+        curriculum (str): The curriculum of the search query (can be null).
+        education_level (str): The education level of the search query (can be null).
+    """
+
     class Meta:
         verbose_name_plural = "Search Queries"
     
-    query = models.CharField(max_length=255, verbose_name="query")
+    query = models.CharField(max_length=255, verbose_name="query", default=None, null=True, blank=True)
```

Comment:
It was done for debugging purposes, to test different scenarios. Have removed "default=None", "null=True" and "blank=True" from query. But have kept these properties in curriculum and education_level, since the model "SearchQuery" is being used as a common model among different APIs. Some APIs required curriculum or education_level as data, and some do not.